src/server/test/web/readingsBarGroupQuantity.js

```diff
@@ -207,8 +207,22 @@ mocha.describe('readings API', () => {
 
                 // Add BG13 here
 
-                // Add BG14 here
+                mocha.it('BG14: 1 day bars for 15 + 20 minute reading intervals and quantity units with +-inf start/end time & kWh as lbs of CO2 & chained & reversed', async () =>{
+                    //loads data into database
```

Comment:
The description lists these units/conversions: u2, u10, u11, u12, u13, c11, c12, c13, c14. All but u2 need to be included in the test. Test B14 uses the same units and is completed.